src/Middlewares/AuthMiddleware.php

```diff
@@ -33,7 +33,7 @@ public function __construct($config)
      */
     public function __invoke($request, $response, $next)
     {
-        if (!empty($this->config->auth)) {
+        if ($this->config->useAuth) {
```

Comment:
Change constructor to only accept the `Method` class, so we can guarantee this exists.